src/components/Books.js

```diff
@@ -1,11 +1,41 @@
-import List from './List';
-import Form from './Form';
-
-const Books = () => (
-  <>
-    <List />
-    <Form />
-  </>
-);
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import Book from './Book';
+import BookForm from './BookForm';
+import { addBook, removeBook } from '../redux/books/booksSlice';
+
+const Books = () => {
+  const dispatch = useDispatch();
+  const books = useSelector((state) => state.books.books);
+
+  const handleAddBook = (title, author) => {
+    const newBook = {
+      item_id: `item${books.length + 1}`,
+      title,
+      author,
+    };
+    dispatch(addBook(newBook));
+  };
+
+  const handleDeleteBook = (bookId) => {
+    dispatch(removeBook(bookId));
+  };
```

Comment:
- [ ] **[OPTIONAL]** Since you're using a global state to handle all operations related to books, there's no need to define these functions inside a parent component (lift the state up). Instead, you could simply move this logic to the component that actually needs it, for example, you can move the `handleAddBook` function to your `BookForm` component and the `handleDeleteBook` to your `Book` component. 

Ending up not having to pass these functions as props here and with a more readable and cleaner code.


e.g.

```js
// src/components/Book.js
const Books = () => {
  const books = useSelector((state) => state.books.books);

  return (
    <>
      <ul className="book-list">
        {books.map((book) => (
          <Book
            key={book.item_id}
            id={book.item_id}
            bookTitle={book.title}
            author={book.author}
          />
        ))}
      </ul>
      <BookForm />
    </>
  );
};
```

```js
// src/components/BookForm.js
const AddBook = () => {
  const dispatch = useDispatch();
  const [title, setTitle] = useState('');
  const [author, setAuthor] = useState('');

  const handleAddBook = () => {
    const newItemId = `item${Math.random().toString(36).substr(2, 9)}`;

    dispatch(addBook({
      item_id: newItemId,
      title,
      author,
      category: 'Fiction',
    }));

    setTitle('');
    setAuthor('');
  };

  return (
    <div>
      <h1>ADD NEW BOOK</h1>
      <form>
        <input
          type="text"
          placeholder="Book Title"
          value={title}
          onChange={(e) => setTitle(e.target.value)}
        />
        <input
          type="text"
          placeholder="Author"
          value={author}
          onChange={(e) => setAuthor(e.target.value)}
        />
        <button onClick={handleAddBook} type="submit">
          Add Book
        </button>
      </form>
    </div>
  );
};
```

```js
// src/components/Book.js
const Book = ({ bookTitle, author, id }) => {
  const dispatch = useDispatch();

  const handleDeleteBook = () => {
    dispatch(removeBook(id));
  };
  
  return(
    <li>
      <h1>{bookTitle}</h1>
      <p>{author}</p>
      <button type="button">Delete</button>
      <div>
        <strong>Title:</strong>
        {' '}
        {bookTitle}
      </div>
      <div>
        <strong>Author:</strong>
        {' '}
        {author}
      </div>
      <button onClick={handleDeleteBook} type="button">
        Delete
      </button>
    </li>
  );
}
```

It would be great if you can apply this change! 🤓 🔧